packages/x-components/src/x-modules/scroll/components/main-scroll-item.vue

```diff
@@ -99,7 +112,9 @@
       oldObserver?.unobserve(this.$el);
       newObserver?.observe(this.$el);
       if (this.pendingScrollTo === this.item.id) {
-        this.$el.scrollIntoView();
+        Vue.nextTick(() => {
+          this.$el.scrollIntoView();
```

Comment:
```suggestion
          this.$el.scrollIntoView({block: 'center'})
```

After doing some testing, this seems a good option. This option centers always the item in the viewport, so we would have some part of the previous row visible, but the specified item will be always in the view port.

Downside: We would have to modify some test probably.